src/BatchFormat/source.extension.vsixmanifest

```diff
@@ -15,19 +15,25 @@ Function: Remove, Sort, Using, Format Document, and more combinations.</Descript
     <InstalledByMsi>false</InstalledByMsi>
     <SupportedProducts>
       <VisualStudio Version="10.0">
-        <Edition>Ultimate</Edition>
         <Edition>Premium</Edition>
         <Edition>Pro</Edition>
+        <Edition>Ultimate</Edition>
       </VisualStudio>
       <VisualStudio Version="11.0">
-        <Edition>Ultimate</Edition>
         <Edition>Premium</Edition>
         <Edition>Pro</Edition>
+        <Edition>Ultimate</Edition>
       </VisualStudio>
       <VisualStudio Version="12.0">
-        <Edition>Ultimate</Edition>
+        <Edition>Community</Edition>
         <Edition>Premium</Edition>
         <Edition>Pro</Edition>
+        <Edition>Ultimate</Edition>
+      </VisualStudio>
+      <VisualStudio Version="14.0">
+        <Edition>Community</Edition>
+        <Edition>Enterprise</Edition>
+        <Edition>Pro</Edition>
       </VisualStudio>
```

Comment:
No problem! And thanks for this handy extension. :rocket: 
Please let me know if you find anything interesting in your review.  :)

Note that it requires VS2015 to build the project because I used C# 6 syntax such as; [string interpolation (`$"{variable}"`)](https://github.com/dotnet/roslyn/wiki/New-Language-Features-in-C%23-6#string-interpolation) and [null-conditional operator (`?.`)](https://github.com/dotnet/roslyn/wiki/New-Language-Features-in-C%23-6#null-conditional-operators)). 

If you want to build using VS2010 - VS2013, you can simply run:

```powershell
Install-Package Microsoft.Net.Compilers
```

in `Package Manger Console` window and it will download Roslyn. This package enables C#6 compilation (I haven't tested with VS2013). Note that in this case, the editor will show the C#6 syntax as invalid syntax but the compiler will continue to work.